dataset/dbengine.py

```diff
@@ -122,7 +122,8 @@ def create_db_table_from_query(self, name, query):
         return True
 
     def create_db_index(self, name, table, attr_list):
-        stmt = index_template.substitute(idx_title=name, table=table, attr=','.join(attr_list))
+        quoted_attrs = list(map('"{}"'.format, attr_list))
```

Comment:
Do you think this is more readable?
`['"{}"'.format(attr) for attr in attr_list]`

Reply:
Yes I agree. Changed.